packages/settings-plugins/settings-plugins.spec

```diff
@@ -0,0 +1,205 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+%global _cross_pluginsdir %{_cross_libdir}/settings-plugins
+
+Name: %{_cross_os}settings-plugins
+Version: 0.0
+Release: 0%{?dist}
+Summary: Settings plugins
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}glibc-devel
+
+Requires: %{_cross_os}settings-plugin(any)
```

Comment:
> I assume we'll need hoist this or something like it into os to force kits to provide something?

I added the dependency to `os.spec` in a later. However, your comment made me realize that in the absence of a "preferred" choice defined by requires in the top-level package, the approach here would allow `dnf` to pick a settings plugin at random to satisfy the `%{_cross_os}settings-plugin(any)` dependency. (And same for defaults.)

I'll tighten that up in the next revision.